components/home/Header/styled.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from "styled-components";
+
+interface ContainerProps {}
+export const Container = styled.div<ContainerProps>`
+  display: flex;
+  flex-direction: column;
+`;
+
+interface NavWrapperProps {}
+export const NavWrapper = styled.div<NavWrapperProps>`
+  background-color: #ff652e;
+  padding: 20px 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
+
+interface SocialMediaLinksProps {}
+export const SocialMediaLinks = styled.div<SocialMediaLinksProps>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  column-gap: 24px;
+  margin: 16px 0;
+`;
+
+interface NavbarProps {}
+export const Navbar = styled.nav<NavbarProps>`
+  width: 820px;
+  height: 40px;
+
+  ul {
+    grid-auto-columns: 1fr;
+    grid-auto-flow: column;
+    column-gap: 2px;
+    padding-left: 0;
+    display: grid;
+    height: 100%;
+    margin: 0;
+
+    li {
```

Comment:
Separate component.

Reply:
Please clarify, do you mean I write separate styled component instead of styling nested tags?

Reply:
each DOM element (div, li, etc) should be a styled component, so that we can use and see it in isolation

Reply:
Got it.